packages/api/src/Base.ts

```diff
@@ -353,6 +354,14 @@ export default abstract class ApiBase<CodecResult, SubscriptionResult> implement
           this._isReady = true;
 
           this.emit('ready', this);
+
+          setInterval(() => {
+            if (this._rpcBase._provider.isConnected()) {
```

Comment:
discard the interval the handler means it is now impossible to destroy this api instance.
should have a close/dispose/teardown method to `clearInterval`

Reply:
100%, still WIP... (Hacked it in for the other tests to work)

Reply:
Should be addressed around here - https://github.com/polkadot-js/api/pull/805/files#diff-95597f67ed68d7dffc27ba1f0501e5b1R336

So `disconnect()` breaks the link, the event from that clears the interval.